clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3269,6 +3279,51 @@ static void handleIntelReqdSubGroupSize(Sema &S, Decl *D,
   S.AddIntelReqdSubGroupSize(D, AL, E);
 }
 
+IntelNamedSubGroupSizeAttr *
+Sema::MergeIntelNamedSubGroupSizeAttr(Decl *D,
+                                      const IntelNamedSubGroupSizeAttr &A) {
+  if (checkAttrMutualExclusion<IntelReqdSubGroupSizeAttr>(*this, D, A))
+    return nullptr;
+  if (checkAttrMutualExclusion<SYCLSimdAttr>(*this, D, A))
```

Comment:
Amusingly, we're not done yet -- you'll need to add a new handler and merge function for `SYCLSimdAttr` to catch the same things. Otherwise you won't catch the conflict with code like:
```
[[intel::named_sub_group_size(automatic)]] void func();
[[intel::sycl_explicit_simd]] void func();
```

Reply:
Ugh, fine...